velox/functions/lib/string/StringImpl.h

```diff
@@ -147,6 +147,36 @@ FOLLY_ALWAYS_INLINE int32_t charToCodePoint(const T& inputString) {
   return codePoint;
 }
 
+/// Returns the sequence of Unicode code point of characters in the an input
+/// string.
+template <bool isAscii, typename T>
+FOLLY_ALWAYS_INLINE std::vector<int32_t> stringToCodePoint(
+    const T& inputString) {
+  std::vector<int32_t> res;
```

Comment:
Reserve vector's capacity. Read https://velox-lib.io/blog/array-sort on why this is important.